osltweaks.php

```diff
@@ -16,6 +16,35 @@ function osltweaks_civicrm_buildForm($formName, &$form) {
       // include some javascript to finagle the wording in that section.
       CRM_Core_Resources::singleton()->addScriptFile('org.osltoday.osltweaks', 'js/CRM_Contribute_Form_Contribution_Main-isShowCMS.js');
     }
+
+    // Array of Contribution page id for the Country and State rewrite
+    $settings = CRM_Core_BAO_Setting::getItem(NULL, 'com.joineryhq.osltweaks');
+
+    foreach ($settings['page_ids'] as $key => $id) {
```

Comment:
Please  rename this as `$settings['us_only_page_ids']`. (Who knows, maybe later we will have another kind of 'page_id' to keep track of in `$settings`, so a more specific name is helpful here.)

Reply:
Instead of a `foreach` loop, please use `if (in_array($form->_id, $settings['us_only_page_ids'])) {`.  The problem with `foreach` is that it will continue to loop through all values of `$settings['us_only_page_ids']`, which is less efficient and not needed. We just need to know if `$form->_id` is included in `$settings['us_only_page_ids']`, so we only need that one `if()`.

Reply:
Done